File: 800/C++/p15.cpp
#include<bits/stdc++.h>
using namespace std;

void solve() {
    int n;
    cin>>n;

    int oddCnt = 0;
    for(int i = 0; i < n; i++) {
        int num;
        cin>>num;

        oddCnt += (num & 1);
    }

    if(oddCnt & 1)  cout<<"No"<<endl;
    else    cout<<"Yes"<<endl;
}

int main() {
    int t;
    cin>>t;
    while(t--)  solve();
    return 0;
}

File: 800/Java/P15.java
import java.util.*;

public class P15 {
    static void solve(Scanner scanner) {
        int n = scanner.nextInt();
        int oddCnt = 0;
        for(int i = 0; i < n; i++) {
            int num = scanner.nextInt();
            if(num % 2 == 1)    oddCnt++;
        }

        if(oddCnt % 2 == 0) System.out.println("Yes");
        else    System.out.println("No");
    } 

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int t = scanner.nextInt();
        while(t-- > 0)  solve(scanner);
        scanner.close();
    }
}